app/components/answer-step.jsx

```diff
@@ -0,0 +1,152 @@
+      _id: startingPoint._id,
+    }
+    setStartingPoint(updatedAnswers)
+  }
+
+  const updateWhyResponse = ({ valid, value }) => {
+    const updatedAnswers = whyMosts.map(answer => {
+      // find which why response to update
+      if (answer._id === value.parentId) {
+        answer.answerSubject = value.subject
+        answer.answerDescription = value.description
+        answer.valid = valid
+      }
+      return answer
+    })
+    setWhyMosts(updatedAnswers)
+  }
+
+  // evaluates if both subject and description fields are filled out for starting point
+  const areStartingComplete = () => {
+    return startingPoint.answerSubject && startingPoint.answerDescription
+  }
+
+  // evaluates if both subject and description fields are filled out for whyMosts
+  const areWhyAnswersComplete = () => {
+    return whyMosts.every(answer => answer.valid)
+  }
+
+  return (
+    <div className={cx(classes.wrapper, className)} {...otherProps}>
+      <StepIntro subject="Answer" description="Please provide a title and short description of your answer." />
+      <div className={classes.answersContainer}>
+        {question ? (
+          <div key={startingPoint._id}>
+            <WhyInput
+              point={{ subject: '', description: question, _id: startingPoint._id }}
+              defaultValue={{ subject: '', description: '' }}
+              onDone={updateQuestionResponse}
+            />
+          </div>
+        ) : (
+          <div className={classes.noPointsContainer}>
+            <hr className={classes.pointsHr}></hr>
+            There are no questions to respond to.
+          </div>
+        )}
+        {whyMosts.length ? (
+          whyMosts.map(point => (
+            <div key={point._id}>
+              <hr className={classes.pointsHr}></hr>
+              <WhyInput
+                point={{ subject: '', description: whyQuestion, _id: point._id }} // _id is parentId
+                defaultValue={{ subject: '', description: '' }}
+                onDone={updateWhyResponse}
+              />
+            </div>
+          ))
+        ) : (
+          <div className={classes.noPointsContainer}>
+            <hr className={classes.pointsHr}></hr>
+            There are no whyQuestion to respond to.
+          </div>
+        )}
+      </div>
+    </div>
+  )
+})
+
+const useStylesFromThemeFunction = createUseStyles(theme => ({
+  wrapper: {
+    background: theme.colorPrimary,
+  },
+  introContainer: {
+    textAlign: 'left',
+    padding: '0 1.875rem',
+  },
+  introTitle: {
+    fontSize: '2.25rem',
+    paddingBottom: '2rem',
+  },
+  introText: {
+    display: 'block',
+    fontSize: '1.25rem',
+  },
+  [`@media (min-width: ${theme.condensedWidthBreakPoint})`]: {
+    introContainer: {
+      padding: '0',
+    },
+    introText: {
+      maxWidth: '33rem',
+    },
+  },
+  answersContainer: {
+    fontSize: '1.25rem',
+    paddingTop: '4.325rem',
+  },
+  pointsHr: {
+    color: '#D9D9D9',
```

Comment:
Please use colores out of theme, if this one isn't already there you can put it there.